### PR TITLE
Rich cards view

### LINKED ISSUE #2: app is currently checking for `sanitized_url` json field.
This field has been changed to just `url` on the magpie side.

### PR DESCRIPTION
**Pull request by AliceYuan**
- handle video, images and no images/video

also resolves #2 
screenshot 2015-07-22 00 39 27
screenshot 2015-07-22 00 42 14
screenshot 2015-07-22 00 42 29

### CODE REVIEW

File: web/app/templates/card.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+					<p >{{ json['provider_url'] }}</p>
+				</div>
+				{% endif %}
+			</div>
+			<div class="left-col">
+				{% if json['media'] and json['media']['data'] and json['media']['data'][0]['iframe'] %}
+					<div class="media"> {{ json['media']['data'][0]['iframe']|safe }}</div>
+				{% elif json['images'] and json['images']['data'] %}
+					<div class="images">
+	<!-- 				{% for i in range(json['images']['data']|length) %}
```
Comment (MeenuV) [inline web/app/templates/card.html:28]: Is this commented out portion required? Or can be deleted?

Comment (AliceYuan) [inline web/app/templates/card.html:28]: incase we want some slider image thing in the future where you can pick the images :p

File: web/app/setup.py [hunk 2]
```diff
@@ -53,9 +53,8 @@ def make_pretty_json(response_json):
         return json_output
 
     def show_response_errors(response_json):
-        print response_json
-        if (response_json['status'] != StatusCode.OK):
-            flash("ERROR: " + response_json['error_message'], 'error')
+        if (response_json['status'] != StatusCode.OK and response_json['error_message']):
```
Comment (MeenuV) [inline web/app/setup.py:56]: Extract `status` and `error_message` to vars?

**Comment by MeenuV** [comment]
+1

**Pull request merged**